forest/cli/src/cli/wallet_cmd.rs

```diff
@@ -54,7 +55,17 @@ pub enum WalletCommands {
         path: Option<String>,
     },
     /// List addresses of the wallet
-    List,
+    List {
+        //flag to show exact balance meaming
```

Comment:
1. This should be a documentation comment, so `///`
2. I cannot really deduce what I should expect from this and the other flag. Users probably will also struggle. Please provide examples.

Reply:
Fixed